lib/g_places.js

```diff
@@ -0,0 +1,83 @@
+https = require('https');
+
+/**
+ * function to search google places
+ * @param  {[object] name, lat, long from input form}
+ * @return {[object] gp - literal with google place values for newRest }
+ */
+module.exports = function(params, callback) {
+
+  var apikey = 'AIzaSyCY1BQZJ8B4_QjNr-UDFGvtInFs9SF7JM0';
```

Comment:
thanks. Do you mean public as in the request?  or as in the code?  We can put it in an ignored file, I guess...